app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actors_cursor]
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} VALUES (NULL, ?, ?)",
+            (first_name, last_name)
```

Comment:
`id` must not be `NULL`. Insert only `first_name` and `last_name` and `id` will appear automatically